alvr/client_core/src/connection.rs

```diff
@@ -479,39 +463,12 @@ fn connection_pipeline(
 
                 match maybe_packet {
                     Ok(ServerControlPacket::DecoderConfig(config)) => {
-                        if capabilities.external_decoder {
-                            event_queue
-                                .lock()
-                                .push_back(ClientCoreEvent::DecoderConfig {
-                                    codec: config.codec,
-                                    config_nal: config.config_buffer,
-                                });
-                        } else if ctx.decoder_sink.lock().is_none() {
```

Comment:
Wait why can this be removed

Reply:
The decoder is instantiated in client_openxr instead of here